go.mod

```diff
@@ -1,12 +1,12 @@
 module github.com/IBM/ibm-vpc-file-csi-driver
 
-go 1.23.8
+go 1.23.10
 
 require (
 	github.com/IBM/ibm-csi-common v1.1.20
-	github.com/IBM/ibmcloud-volume-file-vpc v1.2.9
-	github.com/IBM/ibmcloud-volume-interface v1.2.12
-	github.com/IBM/secret-utils-lib v1.1.13
+	github.com/IBM/ibmcloud-volume-file-vpc v1.2.10-0.20250807172559-a84ce64273a2
+	github.com/IBM/ibmcloud-volume-interface v1.2.14-0.20250730121137-788087caceb2
```

Comment:
Revert this interface version , we can later add file-vpc version but we can merge as of now with reverting interface